bin/mv.js

```diff
@@ -141,31 +138,118 @@ function loadProfile(name) {
     return config.getProfile(name);
 }
 
-function moveCronJob(profile, name, target) {
+function moveCronJob(profile, name, target, options) {
     return coroutine(function*() {
-        var cronJobs = yield profile.listCronJobs();
-
-        for (let job of cronJobs) {
-            if (job.name === name) {
-                let token = yield profile.getWebtask({
-                    name: target.name
-                });
-                yield target.profile.createCronJob({
-                    name: target.name,
-                    container: target.container,
-                    token: token.token,
-                    schedule: job.schedule
-                });
-                yield profile.removeCronJob({
-                    name: name
-                });
-                break;
+        var job;
+
+        try {
+            job = yield profile.getCronJob({
+                name: name
+            });
+        } catch (err) {
+            if (err.statusCode === 404) {
+                return;
             }
+            throw err;
+        }
+
+        let webtask = yield profile.getWebtask({
```

Comment:
I'm a bit confused by this bit here.

So far, the `source` webtask has been recreated as the `target` webtask and now we're seeing if a cron-job also needs to be migrated. To do so, I think the algorithm should look like:

1. Check if there is an existing `source` cron job with the source webtask's `name`.
2. If there is such a cron job, we compare the `source` cron job's token to the `source` webtask's token. (Note that we should already have a copy of the source webtask's token as a result of the `inspect` operation earlier)
3. If the two match, continue as you have done below.

The key difference is recognizing that we already have the source webtask's token from earlier steps. LMK if I've missed something important.